pghoard/pghoard.py

```diff
@@ -297,6 +298,36 @@ def delete_remote_basebackup(self, site, basebackup, metadata):
                             chunk["chunk_filename"],
                         )
                     )
+        elif metadata.get("format") == BaseBackupFormat.delta_v1:
```

Comment:
I wonder could function be more observable if the logic for the `delta_v1` format is extracted into a separate function, so we don't have long functions with multiple levels nesting for the sake of readability.